pkgs/top-level/all-packages.nix

```diff
@@ -29495,7 +29495,7 @@ with pkgs;
   digibyte = libsForQt514.callPackage ../applications/blockchains/digibyte { withGui = true; };
   digibyted = callPackage ../applications/blockchains/digibyte { withGui = false; };
 
-  dogecoin  = callPackage ../applications/blockchains/dogecoin {
+  dogecoin  = libsForQt5.callPackage ../applications/blockchains/dogecoin {
     boost = boost165;
     withGui = true;
   };
```

Comment:
There is `dogecoind` on the next line, it needs `libsForQt5.callPackage` as well

Reply:
`dogecoind` is just a daemon without GUI, so it shouldn't require this. It's the same situation as with the `bitcoin` and `bitcoind`

Reply:
Qt is not only a GUI library